### PR TITLE
Feature update schema docs

### PR DESCRIPTION
**Pull request by wadeking98**
added documentation to direct issuers on how to update credential schemas to support schema labels, highlighted attributes, credential titles, etc

### CODE REVIEW

**Comment by WadeBarnes** [comment]
Please rebase your changes on the latest from the main branch to eliminate the merge from your commit history.

File: docs/README.md [hunk 1]
```diff
@@ -206,6 +207,9 @@ The DID is located in the `aries-vcr-issuer-controller/docker/manage` file on li
 
 Navigate to the docker folder in the project. Run: `./manage build` to build all the necessary containers. Finally run `./manage start` to initialize the project.
 
+### Updates to Credential Schema
+Aries-vcr has undergone updates to the issuer credential schema. See the latest updates and how to apply them [here](Schema-changes.md)
```
Comment (WadeBarnes) [inline docs/README.md:211]:
```suggestion
Aries-vcr has undergone updates to the issuer credential schema. See the latest updates and how to apply them [here](Schema-changes.md).
```

File: docs/README.md [hunk 2]
```diff
@@ -20,6 +20,7 @@
   - [Initialize the Project](###initialize-the-project)
   - [Register the DID](###register-the-did)
   - [Build and Start the Project](###build-and-start-the-project)
+  - [Updates to Credential Schema](###updates-to-credential-schema)
```
Comment (WadeBarnes) [inline docs/README.md:23]: The table of content links (all of them) are broken.  I believe the proper syntax for a TOC link is to use a single `#`.  Please fix.

File: docs/Schema-changes.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
+### Changes to Aries-vcr issuer schema
+
+The Aries-vcr application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are issued.
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:3]:
```suggestion
The Aries VCR application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are processed.  
```

Please replace 'Aries-vcr' with 'Aries VCR' throughout to keep things consistent with the rest of the documentation.

File: docs/Schema-changes.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
+### Changes to Aries-vcr issuer schema
+
+The Aries-vcr application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries-vcr application is in the way that credential schemas are issued.  
+  
+Included below are a list of optional changes to credential schemas that issuers may be interested in making. For the purpose of clearly displaying how these changes are being used, we've provided examples of the credential changes in the new Orgbook site.
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:5]: "we've provided examples of the credential changes in the new Orgbook site."  Provide a link to the repository so it is clear what code is being used for the examples.

File: docs/Schema-changes.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+"highlighted_attributes":[
+	"<YOUR HIGHLIGHTED ATTRIBUTE KEY 1>",
+	"<YOUR HIGHLIGHTED ATTRIBUTE KEY 2>"
+  ],
+"credential_title":"<YOUR CREDENTIAL TITLE KEY>",
+~~~
+highlighted attribute keys and the credential title key refer to the attribute names on each of the attributes in the schema. An example is shown in figure 4.1. Highlighted attributes will appear on credential cards in the order they're listed on the schema. Example: attribute #1 will appear above attribute #2 on this credential card.
+
+![](./assets/attribute_key.png)
+*Figure 4.1: Example attribute key*
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:145]: Figure title text is displaying inline with the figure rather than under it.

File: docs/Schema-changes.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+
+##### Setting the credential title and highlighted attribute labels:
+Attribute labels appear to the left of an attribute value on a credential card. The attribute label describes the attribute value being displayed on the credential card. See figure 4.2 for an example.
+
+![](./assets/attribute_label_value.png)
+*Figure 4.2: Attribute label and value*
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:151]: Figure title text is displaying inline with the figure rather than under it.

File: docs/Schema-changes.md [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+
+To set the attribute label simply edit the claim label for the attribute in question. To do so, in `metadata->labels->attributes->translations->[locale]->label` change the `name` field to your desired label as seen in figure 4.3 where we change the `reason_description` from `Reason` to `DEMO`. See figure 4.4 for the resulting change in Orgbook
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:153]: Consider using the more standard `.` notation for describing the path in the json (throughout the document).  For example:
- `metadata->labels->attributes->translations->[locale]->label` should be `metadata.labels.attributes.translations.[locale].label`

**Review by WadeBarnes — CHANGES_REQUESTED** [review]

File: docs/Schema-changes.md [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+  "schema_version": "1.0.44",
+  "attributes": [
+    "registration_id",
+    "registration_date",
+    ...
+    "reason_description",
+    "expiry_date"
+  ],
+  "metadata": {
+    "topic": [
```
Comment (ianco) [inline docs/Schema-changes.md:48]: This message structure doesn't match the structure passed to the `/registration` endpoint nor the structure of the message passed from one agent to the other as part of the registration protocol.

Is this the message structure internal to the new Issuer Agency?

**Review by ianco — CHANGES_REQUESTED** [review]
Docs should match against the registration message structure

File: docs/Schema-changes.md [hunk 9]
```diff
@@ -0,0 +1,176 @@
+### Changes to Aries VCR issuer schema
+
+The Aries VCR application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries VCR application is in the way that credential schemas are issued.
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:3]:
```suggestion
The Aries VCR application is undergoing changes to enhance user experience on applications such as Orgbook. The primary change to the Aries VCR application is in the way that credential schemas are processed.  
```

File: docs/Schema-changes.md [hunk 10]
```diff
@@ -0,0 +1,176 @@
...
+
+![](./assets/highlighted_attributes.png)  
+*Figure 2: Highlighted attributes*
+
+#### Schema labels
+Aries VCR has added the functionality to specify the name of a credential schema in a multilingual format. These schema names appear as the title of the credential and let users know what type of credential they are viewing. This schema label is optional and is only needed if you plan on supporting multi language use. If no schema label is defined, a good fallback is to use the credential type description displayed in the same way as a schema label. The new version of Orgbook takes this approach. See figure 1 for example of a schema label in Orgbook.
+  
+#### Aries VCR defaults
+##### Credential title:
+If the credential title is not specified in a credential schema, or there is no claim label to match the credential title, then applications such as Orgbook may instead use the registration number of the entity holding the credential. See figure 3 for an example
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:24]:
```suggestion
If the credential title is not specified in a credential schema, or there is no claim label to match the credential title, then applications such as Orgbook may instead use the registration number of the entity holding the credential. See figure 3 for an example.
```

File: docs/Schema-changes.md [hunk 11]
```diff
@@ -0,0 +1,176 @@
...
+![](./assets/default_cred_title.png)  
+*Figure 3: default credential title and highlighted attributes*
+  
+##### Highlighted attributes:
+If no highlighted attributes are specified, or there are no claim labels to match the highlighted attributes, then applications like Orgbook will not display any highlighted attributes. See figure 3.1 to see how a credential with no highlighted attributes or credential title is displayed.  
+![](./assets/default_highlighted.png)  
+*Figure 3.1: Default credential title and highlighted attributes*
+
+#### Recommended changes to schema
+To incorporate these optional changes, Aries VCR recommends reviewing the new schema attributes and how to add them to your credential schema in the section below. The following example JSON snippet is from the `issuer_registration.credential_types[item]` section of the registration.registries.ca schema
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:34]:
```suggestion
To incorporate these optional changes, Aries VCR recommends reviewing the new schema attributes and how to add them to your credential schema in the section below. The following example JSON snippet is from the `issuer_registration.credential_types[item]` section of the registration.registries.ca schema.
```

File: docs/Schema-changes.md [hunk 12]
```diff
@@ -0,0 +1,176 @@
...
+      }
+    }
+  ],
+  "version": "1.0.42"
+}
+~~~
+*Figure 4: Example registration schema*
+
+##### Setting the schema label:
+In order to set the schema label add the following JSON object to `issuer_registration.credential_types[item]`
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:134]:
```suggestion
In order to set the schema label add the following JSON object to `issuer_registration.credential_types[item]`:
```

File: docs/Schema-changes.md [hunk 13]
```diff
@@ -0,0 +1,176 @@
...
+~~~
+"labels": {
+	"en": "<YOUR SCHEMA NAME, Example: Registration >"
+}
+~~~
+
+
+##### Setting the credential title and highlighted attributes:
+To set the credential title and highlighted attributes setup Aries VCR issuer controller and add the following YAML object to `issuers.myorg.credential_types[index]` in `services.yaml`. Figure 4.1 shows where to insert the yaml object into the services file. Make sure to do this for each
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:143]: `Make sure to do this for each`

Please complete the sentence.

File: docs/Schema-changes.md [hunk 14]
```diff
@@ -0,0 +1,176 @@
...
+![](./assets/yaml_meta_insert.png)  
+*Figure 4.1: Example insertion point*
+
+~~~
+highlighted_attributes:
+- "<YOUR HIGHLIGHTED ATTRIBUTE KEY 1>"
+- "<YOUR HIGHLIGHTED ATTRIBUTE KEY 2>"
+credential_title:"<YOUR CREDENTIAL TITLE KEY>",
+~~~
+highlighted attribute keys and the credential title key refer to the attribute names on each of the attributes in the schema. An example is shown in figure 4.2. Highlighted attributes will appear on credential cards in the order they're listed on the schema. Example: attribute #1 will appear above attribute #2 on this credential card.
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:154]:
```suggestion
Highlighted attribute keys and the credential title key refer to the attribute names on each of the attributes in the schema. An example is shown in figure 4.2. Highlighted attributes will appear on credential cards in the order they're listed on the schema. Example: attribute #1 will appear above attribute #2 on this credential card.
```

File: docs/Schema-changes.md [hunk 15]
```diff
@@ -0,0 +1,176 @@
...
+![](./assets/attribute_key.png)  
+*Figure 4.2: Example attribute key*
+
+##### Setting the credential title and highlighted attribute labels:
+Attribute labels appear to the left of an attribute value on a credential card. The attribute label describes the attribute value being displayed on the credential card. See figure 4.3 for an example.
+
+![](./assets/attribute_label_value.png)  
+*Figure 4.3: Attribute label and value*
+
+To set the attribute label simply edit the claim label for the attribute in question. To do so in your credential schema, go to `issuer_registration.claim_labels.<YOUR ATTRIBUTE KEY>.[locale]` change the field to your desired label as seen in figure 4.4 where we change the `reason_description` from `Reason` to `DEMO`. See figure 4.5 for the resulting change in Orgbook
```
Comment (WadeBarnes) [inline docs/Schema-changes.md:165]:
```suggestion
To set the attribute label simply edit the claim label for the attribute in question. To do so in your credential schema, go to `issuer_registration.claim_labels.<YOUR ATTRIBUTE KEY>.[locale]` change the field to your desired label as seen in figure 4.4 where we change the `reason_description` from `Reason` to `DEMO`. See figure 4.5 for the resulting change in Orgbook.
```

**Review by WadeBarnes — CHANGES_REQUESTED** [review]
A few small changes.

@ianco, could you review the schema example again please?

File: docs/Schema-changes.md [hunk 16]
```diff
@@ -0,0 +1,176 @@
...
+  
+##### Highlighted attributes:
+If no highlighted attributes are specified, or there are no claim labels to match the highlighted attributes, then applications like Orgbook will not display any highlighted attributes. See figure 3.1 to see how a credential with no highlighted attributes or credential title is displayed.  
+![](./assets/default_highlighted.png)  
+*Figure 3.1: Default credential title and highlighted attributes*
+
+#### Recommended changes to schema
+To incorporate these optional changes, Aries VCR recommends reviewing the new schema attributes and how to add them to your credential schema in the section below. The following example JSON snippet is from the `issuer_registration.credential_types[item]` section of the registration.registries.ca schema
+
+~~~
```
Comment (ianco) [inline docs/Schema-changes.md:36]: This looks good.  Probably worth pointing out where this format of message is used (is it the format provided to the new issuer agency?) and linking to the demo issuer/controller (https://github.com/bcgov/aries-vcr-issuer-controller/tree/main/issuer_controller/config#changes-for-schema-localization), which has applied a corresponding set of changes, and represents the changes that the current set of issuers will need to make.

**Review by ianco — CHANGES_REQUESTED** [review]

**Comment by ianco** [comment]
Some commits are missing DCO signoff

**Review by WadeBarnes — APPROVED** [review]

**Review by ianco — APPROVED** [review]

**Pull request merged**